### ISSUE TITLE
run a subset of methods over a seed parameter grid

### ISSUE THREAD

**Issue by btraven00** [omnibenchmark/omnibenchmark_paper_code#4]
# Definition of done
* [x] we document a clustering yaml to run a grid of:
```
dataset x method x seed x metric x k 
```
* [x] we provide scripts to process a collection of output folders and generate a data frame for further analysis 

## Pre-requisites: #3

**Comment by btraven00** [comment]
[metric-performance__202506241005.csv](https://github.com/user-attachments/files/20878856/metric-performance__202506241005.csv)
[method-performance__202506241005.csv](https://github.com/user-attachments/files/20878855/method-performance__202506241005.csv)

Leaving here two dataframes from an experiment, for analysis and quality check.

- 2 backends (apptainer, conda) - easybuild env is currently not building.
- 6 datasets (from[ imallona branch](https://github.com/omnibenchmark/clustering_example/pull/30) at the latest in the PR)
- 9 methods
- 5 seeds
- 7 metrics

These method/dataset combinations behave quite nicely, so it's quite feasible to add more repetitions.

Yamls can be generated from the files in #5, and aggregation script is found in #6 

Code this was run using https://github.com/omnibenchmark/omnibenchmark/tree/devplusplus
(--out-dir, --local-timeout, and profiler over last release).

**Comment by btraven00** [comment]
Just floating the idea: parameter grids are ugly and error prone. Lots of copy and paste. Wecould consider generating them with a custom combinatorial directive (https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.ParameterGrid.html, or the github yaml matrix)
